generators/client/templates/react/src/main/webapp/app/config/translation.ts

```diff
@@ -1,20 +1,32 @@
 /* global: require*/
 import { TranslatorContext } from 'react-jhipster';
+import * as _ from 'lodash';
 
 import { setLocale } from '../reducers/locale';
 import Storage from '../shared/util/storage-util';
 
-const mergeTranslations = requireContext => requireContext.keys().reduce(
-  (merged, key) => ({ ...merged, ...requireContext(key) }),
-  {}
-);
+// const mergeTranslations = requireContext => requireContext.keys().reduce(
```

Comment:
why are these changed? it has nothing to do with entities and I dont see why this change is needed as it will produce some weird code, please revert back to original code

Reply:
I changed this because when I generated several entities, only one file myEntityX.json (with the traduction) was used, the others were ignored :(

Reply:
it should fetch all files and merge them, as its doing the same currently and it works, don't worry about this just take care of entities and we can look into this later